content/blog/python-none/index.md

```diff
@@ -0,0 +1,58 @@
+```markdown
+---
+{
+    title: 'Python None',
+    description: 'Interpreted Languages And Their Footguns',
+    published: '2022-07-27T20:00:00.945Z',
+    edited: '2022-07-27T20:00:00.945ZZ',
+    authors: ['williamcook'],
+    tags: ['python', 'go', 'bugs'],
+    attached: [],
+    license: 'cc-by-nc-nd-4'
```

Comment:
Because this was posted elsewhere originally, we want to make sure to mark it as `originalLink` in order to both:

1) Improve the SEO for your original site
2) Stop Google from marking our site as spam

```suggestion
    license: 'cc-by-nc-nd-4',
    originalLink: "https://williamgeorgecook.com/posts/python_none/"
```